libntfs-3g/fsck.c

```diff
@@ -0,0 +1,155 @@
+#ifdef HAVE_CONFIG_H
+#include "config.h"
```

Comment:
Is there any reason why fsck codes move to library ?

Reply:
because library functions should call function in fsck.c, like as ntfs_cluster_alloc() / ntfs_cluster_free().
those functions set bitmap on disk.
if those were called in fsck context, they should set fsck bitmap also to synchronize bitmap between on disk bitmap and fsck in memory bitmap.